### ISSUE TITLE
Sharable Recipe

### ISSUE THREAD

**Issue by lime62** [beatsbucket/mashub#4]
We should consider recipes to be sharable.

**Comment by wingman7** [comment]
How about just 'Recipe template'. I think we don't need to share it. Just open template to public and someone use it. And it will be independent from core service.

**Comment by lime62** [comment]
Yes, kind of template. Users can create their own recipes using templates and it should be possible their own recipes(which are not from templates) to be templates. And template may need a kind of configuration.
